spec/acceptance/suites/rpm_el7/01_simp_server_spec.rb

```diff
@@ -89,16 +47,16 @@ def internet_yumrepos(host, reponame)
     masters.each do |master|
       it 'should set up SIMP repositories' do
         master.install_package('epel-release')
-
-        tarball = find_tarball
-        if tarball.nil? or tarball.empty?
-          internet_yumrepos(master, find_reponame)
-        else
-          tarball_yumrepos(master, tarball)
-        end
+        setup_repo(master)
         on(master, 'yum makecache')
       end
 
+      use_puppet_repo = ENV['BEAKER_puppet_repo'] || false
+
+      if use_puppet_repo
+        master.install_package('http://yum.puppetlabs.com/puppetlabs-release-pc1-el-7.noarch.rpm')
```

Comment:
This should use the existing capabilities in the libraries.

Reference: https://github.com/simp/rubygem-simp-beaker-helpers/blob/master/lib/simp/beaker_helpers.rb#L588-L598